fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsProductsApiResourceSwagger.java

```diff
@@ -226,88 +226,34 @@ public static final class GetSavingsProductsProductIdResponse {
 
         private GetSavingsProductsProductIdResponse() {}
 
-        static final class GetSavingsProductsAccountingMappings {
-
-            private GetSavingsProductsAccountingMappings() {}
-
-            static final class GetSavingsProductsSavingsReferenceAccount {
-
-                private GetSavingsProductsSavingsReferenceAccount() {}
-
-                @Schema(example = "12")
-                public Integer id;
-                @Schema(example = "savings ref")
-                public String name;
-                @Schema(example = "20")
-                public Integer glCode;
-            }
-
-            static final class GetSavingsProductsIncomeFromFeeAccount {
-
-                private GetSavingsProductsIncomeFromFeeAccount() {}
-
-                @Schema(example = "16")
-                public Integer id;
-                @Schema(example = "income from savings fee")
-                public String name;
-                @Schema(example = "24")
-                public Integer glCode;
-            }
-
-            static final class GetSavingsProductsIncomeFromPenaltyAccount {
-
-                private GetSavingsProductsIncomeFromPenaltyAccount() {}
-
-                @Schema(example = "17")
-                public Integer id;
-                @Schema(example = "income from sav penalties")
-                public String name;
-                @Schema(example = "25")
-                public Integer glCode;
-            }
-
-            static final class GetSavingsProductsInterestOnSavingsAccount {
+        static final class GetSavingsProductsGlAccount {
 
-                private GetSavingsProductsInterestOnSavingsAccount() {}
-
-                @Schema(example = "15")
-                public Integer id;
-                @Schema(example = "interest on savings")
-                public String name;
-                @Schema(example = "23")
-                public Integer glCode;
-            }
+            private GetSavingsProductsGlAccount() {}
 
-            static final class GetSavingsProductsSavingsControlAccount {
-
-                private GetSavingsProductsSavingsControlAccount() {}
-
```

Comment:
I see, however if anyone was using the fineract-client to connect to fineract, now it would be a breaking change for them. Please revert this changes, and write a letter to dev mail list about the changes and the reasoning and let them be aware this is breaking change for anyone who is using fineract-client!